components/ui/popover/BasicPopover.tsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import { Modal, Text, View } from 'react-native';
+
+import { Loader } from '@/components';
+
+interface BasicPopoverProps {
+  title: string;
+  description: string;
+  middleSection?: React.ReactNode;
+  bottomSection?: React.ReactNode;
+  isLoading?: boolean;
+}
+
+const BasicPopover: React.FC<BasicPopoverProps> = ({
+  title,
+  description,
+  middleSection,
+  bottomSection,
+  isLoading = false,
+}) => {
+  return (
+    <Modal transparent>
+      <Loader isLoading={isLoading} />
+      <View className=" w-full h-full flex-col justify-center items-center">
+        <View className="absolute w-full h-full opacity-70 bg-ink-darkest" />
+        <View className="bg-sky-lightest dark:bg-ink-darker w-[327px] min-h-[300px] p-[24px] justify-between rounded-2xl space-y-10">
```

Comment:
do we want to have that as 327px we dont want to have that as more responsive value as rem, case for phone with big screens or very small ?

Reply:
fixed